tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -205,6 +205,8 @@ func {{ template "insertFunctionName" $schema }}({{ if eq (len $schema.Children)
         pgutils.NilOrTime({{$field.Getter "obj"}}),
         {{- else if eq $field.SQLType "uuid" }}
         pgutils.NilOrUUID({{$field.Getter "obj"}}),
+        {{- else if eq $field.DataType "map" }}
+        pgutils.EmptyOrMap({{$field.Getter "obj"}}),
```

Comment:
I think the copyFrom template needs this change too (at least to be future-proof).
Maybe the `{{- range $field := $schema.DBColumnFields -}} ... {{- end }}` pattern could be extracted into another template.

Reply:
oh, good point. I'll try to add a test for it too

Reply:
- https://github.com/stackrox/stackrox/pull/7108